devices/vimar.js

```diff
@@ -42,10 +56,12 @@ module.exports = [
         model: '14593',
         vendor: 'Vimar',
         description: '16A outlet IoT connected',
-        extend: extend.switch(),
+        fromZigbee: [fz.on_off, fz.ignore_basic_report, fzLocal.vimar_active_power],
```

Comment:
Why not use `fz.electrical_measurement` instead of `fzLocal.vimar_active_power`?

Reply:
Because the value reported is wrong, it seems is 10x the real one. We discussed [here](https://github.com/Koenkk/zigbee2mqtt/issues/15482).

Reply:
You can manually set the divisor by adding the following code to `configure`:

```js
            endpoint.saveClusterAttributeKeyValue('haElectricalMeasurement', {
                acPowerMultiplier: 1, acPowerDivisor: 10,
            });
```

Reply:
Ah nice. Let me give it a try. Let you know later.

Reply:
I installed a brand new version of HA and the standard config works correctly, without setting `acPowerMultiplier` and `acPowerDivisor`. I added a commit to change this.